addons/l10n_ec/data/template/account.account-ec.csv

```diff
@@ -1,4 +1,6 @@
 "id","code","name","reconcile","account_type","name@es"
+"ec11010201","11010201","Outstanding Receipts","True","asset_current","Recibos pendientes"
+"ec11010202","11010202","Outstanding Payments","True","asset_current","Pagos pendientes"
```

Comment:
@vbe-odoo I guess bank and bank suspense account will be right after instead of before these then.  That is ok?

Reply:
Yeah... I guess so, can please confirm @kenluzuriaga0 @pepetreshere for this one?
I see that in 17.4 the order and codes were slightly different, see the screenshots below:
**V18 from this PR**
![image](https://github.com/user-attachments/assets/2f6f1daf-dd0b-44fa-bb11-1aaed95bfbe3)

**V17.4 in runbot**
![image](https://github.com/user-attachments/assets/3b6b2d1f-11ed-4ebf-89d0-2eb06a7eb5fb)

Reply:
@pepetreshere @kenluzuriaga0 please help us check the accounts one more time to see which will be the best code and order for them.
Thanks!